### ISSUE TITLE
Namespace assets?

### ISSUE THREAD

**Issue by mightypenguin** [conradz/npm-assets#3]
If this is (or should be) intended to allow npm to handle static assets elegantly like bower, shouldn't this namespace the static dependencies?

i.e. instead of throwing everything into one big directly together, why not put the name of the module as the directory? This would possibly need to account for versions as well if you flattened the tree.

public/assets/<npm_module1>-0.0.1/logo.png
public/assets/<npm_module2>-1.2.3/logo.png

Once a project get's to any size at all I would think you'd have filename conflicts almost guaranteed otherwise.

**Comment by keithamus** [comment]
Good idea, PR welcome - perhaps behind a flag though? Maybe `--namespaced` or something?
